File: dash_docs/chapters/external_resources/examples/local-css.py
import dash
import dash_core_components as dcc
import dash_html_components as html

app = dash.Dash(__name__)

app.layout = html.Div([
    html.Div(
        className="app-header",
        children=[
            html.Div('Plotly Dash', className="app-header--title")
        ]
    ),
    html.Div(
        children=html.Div([
            html.H5('Overview'),
            html.Div('''
                This is an example of a simple Dash app with
                local, customized CSS.
            ''')
        ])
    )
])

if __name__ == '__main__':
    app.run_server(debug=True)

File: dash_docs/chapters/external_resources/examples/local-css.R
library(dash)
library(dashCoreComponents)
library(dashHtmlComponents)

app <- Dash$new()

app$layout(
  htmlDiv(
    list(
      htmlDiv(
        className = "app-header",
        children = list(
          htmlDiv('Plotly Dash', className = "app-header--title")
        )
      ),
      htmlDiv(
        list(
          htmlH5('Overview'),
          htmlDiv(" 
                This is an example of a simple Dash app with
                local, customized CSS.
            ")
        )
      )
    )
  )
)

#app$run_server()
